pages/contact-info/edit.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import { Layout } from '../../components/Layout'
+import { Input } from '../../components/Forms/Input'
+import Select from 'react-select'
+import { NavButtons } from '../../components/NavButtons'
+import { useRouter } from 'next/router'
+import countryList from 'react-select-country-list'
+import { ErrorLabel } from '../../components/Forms/validation/ErrorLabel'
+import { SubmitMessage } from '../../components/Forms/validation/SubmitMessage'
+import { useInternationalization } from '../../components/Hooks'
+import { CommonContactInfo } from './commonContactInfo'
+
+const RESIDENTIAL_ADDRESS = 1
+const MAILING_ADDRESS = 2
+
+export default function Edit({ data }) {
+  return <CommonContactInfo data={data} isDisabled={false}></CommonContactInfo>
```

Comment:
Is testing coming in a future PR? If you'd like an explanation of them you can check out the eligibility estimator front end tests

Reply:
I did not get a chance for writing test. As far as I was told we need few tests only for POC but I do want to add tests in a new PR.